### PR TITLE
docs: Formula Based Quality Inspection

### PR DESCRIPTION
**Pull request by marination**
Docs for https://github.com/frappe/erpnext/pull/23916

- Updated Images and Pre requisites section
- Added Features section
 screencapture-develop-in-8004-docs-user-manual-en-stock-quality-inspection-2020-11-18-15_25_47

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md [hunk 1]
```diff
@@ -4,30 +4,46 @@
...
-Before creating and using a Quality Inspection, it is advised that you create the following first:
+Before creating and using a Quality Inspection, it is advised that you do the following first:
 
-* [Item](/docs/user/manual/en/stock/item)
-* Enable Quality Inspection Criteria in the Item master:
-    ![Enable Quality Inspection](/docs/assets/img/stock/enable-quality-inspection.png)
+* **Create an [Item](/docs/user/manual/en/stock/item)**.
+* **Enable Quality Inspection Criteria in the Item master**. On enabling either checkboxes, **submission** of a stock delivery/receipt document will be allowed only after a Quality Inspection is done against it:
+    ![Enable Quality Inspection](/docs/assets/img/stock/quality-inspection-pre-requisite.png)
+* (Optional) **Create a Quality Inspection Template**. You can add inspection parameters and acceptance criteria in the template, which can be easily fetched into any Quality Inspection. After saving the template, you can set this template in the Item Master (as shown above).
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md:16]: you can set this template in the Item Master (as shown above).

where above?

File: erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md [hunk 2]
```diff
@@ -4,30 +4,46 @@
...
+1. Save. Set the Status. Submit.
 
 <img class="screenshot" alt="Quality Inspection" src="{{docs_base_url}}/assets/img/stock/quality-inspection.png">
 
-## 3. Video
+## 3. Features
+### 3.1 Formula Based Quality Checks
+The acceptance of a parameter/check, in a Quality Inspection, can depend on a certain formula in many cases. The Readings table has a field called **Acceptance Criteria Formula** where you can specify a formula that determines whether a certain check is Accepted or Rejected.
+
+<img class="screenshot" alt="Acceptance Criteria Formula" src="{{docs_base_url}}/assets/img/stock/acceptance-criteria-formula.png">
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md:36]: reading 2, reading 5 and reading 8  boxes are not clear in the image.. either include them completely or don't..

File: erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md [hunk 3]
```diff
@@ -4,30 +4,46 @@
...
+
+This formula depends on the many Reading fields in the Readings table. It can be set manually or via a template.
+
+After setting it, update the readings and Save. The status, in the Readings table rows, is set automatically based on the formula for acceptance.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md:40]:
```suggestion
After setting it, update the readings and Save. The Status field in the Readings table rows is set automatically based on the formula for acceptance.
```

**Review by michellealva — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md (hunk 1 above)
Comment (marination) [reply erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md:16]: @michellealva there is a screenshot right above it

Comment (michellealva) [reply erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md:16]: @marination doesn't show item master.. the screenshot is of the template.. hence, confusing..

Comment (michellealva) [reply erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md:16]: the screenshot above is of the template and not the item master, hence it is confusing..

**Pull request merged**